BUILD.api

```diff
@@ -118,3 +118,55 @@ genrule(
     message = "Applying overrides to cfg proto",
     tools = ["@org_golang_x_tools_imports//:goimports"],
 )
+
+filegroup(
+    name = "mixer/v1/attributes_file",
+    srcs = ["mixer/v1/global_dictionary.yaml"],
+    visibility = ["//visibility:private"],
+)
+
+genrule(
+    name = "mixer/v1/attributes_list",
+    srcs = [":mixer/v1/attributes_file"],
+    outs = ["attributes_list.txt"],
+    cmd = "grep -e '^- ' $(location :mixer/v1/attributes_file) | sed -e 's/- /\"/g' -e 's/$$/\",/g' > $@",
+    message = "Extracting strings",
+    visibility = ["//visibility:private"],
+)
+
+PRE = """package attributes; func GlobalList() ([]string) { tmp := make([]string, len(globalList)); copy(tmp, globalList); return tmp;}; var ( globalList = []string{"""
```

Comment:
it certainly seems possible to create some python to write the source before using bazel to run goimports over it.

would you be opposed to handling that in a follow-on CL (with an issue tracking it)?